crates/oxc_semantic/src/builder.rs

```diff
@@ -250,16 +250,19 @@ impl<'a> SemanticBuilder<'a> {
     }
 
     fn record_ast_nodes(&mut self) {
```

Comment:
this should be `record_ast_node` now. Maybe we can rename the other method to something like `push_ast_record`?

Reply:
@Boshen Feel free to resolve my nitpicks. None of my comments are a blocking change, Just suggestions.